Example/Example/Pods-Example.debug.xcconfig

```diff
@@ -0,0 +1,22 @@
+ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES
```

Comment:
Wondering if we need to check this file in ? I think its generated from cocoapods.